static/home/js/browser_functions.js

```diff
@@ -348,6 +348,15 @@ var tableToExcel = function () {
                 }
             });
 
+            // Convert numbers with "." decimal to numbers with "," so the excel file will have the proper number format
+            $("#excel_table tbody tr").each(function(i, row) {
```

Comment:
This is a fix for local Excel configurations. I would propose to stick with the international/US/UK annotation and keep the dot as decimal.

Reply:
reversed